### PR TITLE
[Remove Stream/Optional usage] LPS-174920 Remove Stream<> usage for module: portal-workflow-kaleo-designer-web

### PR DESCRIPTION
**Pull request by haydenzheng314**
Ticket: https://issues.liferay.com/browse/LPS-174920
Thanks

### CODE REVIEW

**Comment by zhangyan0701** [comment]
ci:test:workflow

**Comment by vicnate5** [comment]
No regressions found.

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/java/com/liferay/portal/workflow/kaleo/designer/web/internal/portlet/KaleoDesignerPortlet.java [hunk 1]
```diff
@@ -361,21 +361,12 @@ private void _serveScriptLanguages(
...
-		List<Object> sortedSupportedScriptLanguages =
-			supportedScriptLanguagesStream.sorted(
-			).collect(
-				Collectors.toList()
-			);
-
 		JSONArray jsonArray = _jsonFactory.createJSONArray();
 
-		for (Object supportedScriptLanguage : sortedSupportedScriptLanguages) {
+		for (Object supportedScriptLanguage :
```
Comment (dantewang) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/java/com/liferay/portal/workflow/kaleo/designer/web/internal/portlet/KaleoDesignerPortlet.java:366]: @LisztLiszt It's already a `List<String>`, so it's redundant to declare the element as `Object` and then `toString()`.

Comment (dantewang) [reply modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/java/com/liferay/portal/workflow/kaleo/designer/web/internal/portlet/KaleoDesignerPortlet.java:366]: @LisztLiszt This for-loop can be simplified replaced with `JSONUtil.toJSONArray(Collection, UnsafeFunction, Log)`.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by dantewang** [comment]
@LisztLiszt Please see the comment and resend, thanks.

**Pull request closed without merging**